src/styles/blocks/catalog.scss

```
@@ -18,6 +18,13 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: transform $duration;
```

```suggestion
    transition: transform $duration, color $duration;
```
you forget `,` that's why transition didn't work last time